dicodile/workers/reusable_workers.py

```diff
@@ -33,47 +29,29 @@ def __init__(self, n_workers, hostfile):
 
     def __del__(self):
         if not self.shutdown:
-            shutdown_reusable_workers()
+            self.shutdown_reusable_workers()
+
+    def send_command(self, tag, verbose=0):
```

Comment:
It would be nice to add some doc string to these methods?

Reply:
yes, and also maybe drop the leading underscore from methods that are used outside of the class?

Reply:
Yes this seems like a nice idea.

Reply:
Oh actually I was confused, there are a few of those in `dicod.py` but not in the `MPIWorkers` class.